src/security-frontend/src/app/common/auth.service.js

```diff
@@ -2,15 +2,25 @@
 	'use strict';
 
 	angular
-			.module('securityManagement')
-			.service('AuthService', AuthService);
+			.module('crm.common')
+			.factory('authService', authService);
 
 	/** @ngInject */
-	function AuthService($http, $q, $log) {
+	function authService($http, $q, $log) {
 		var service = this;
-
 		service.authentication = null;
 
+		return {
+			login: login,
+			logout: logout,
+			restore: restore,
+			isAuthenticated: isAuthenticated,
+			getAuthentication: getAuthentication,
```

Comment:
Think over the above
isAdmin/isManager/isSpecialist methods can both encourage user implement security checks within modules, which are not responsible for this, and create difficulties with maintanability